docs/ops/deployment/kubernetes.md

```diff
@@ -76,57 +74,92 @@ You can then access the Flink UI via different ways:
     {% highlight bash %}./bin/flink run -m localhost:8081 ./examples/streaming/WordCount.jar{% endhighlight %}
 
 *  Create a `NodePort` service on the rest service of jobmanager:
-    1. Run `kubectl create -f jobmanager-rest-service.yaml` to create the `NodePort` service on jobmanager. The example of `jobmanager-rest-service.yaml` can be found in [appendix](#session-cluster-resource-definitions).
+    1. Run `kubectl create -f jobmanager-rest-service.yaml` to create the `NodePort` service on jobmanager. The example of `jobmanager-rest-service.yaml` can be found in [appendix](#common-cluster-resource-definitions).
     2. Run `kubectl get svc flink-jobmanager-rest` to know the `node-port` of this service and navigate to [http://&lt;public-node-ip&gt;:&lt;node-port&gt;](http://<public-node-ip>:<node-port>) in your browser.
-    3. Similarly to `port-forward` solution, you could also use the following command below to submit jobs to the cluster:
+    3. If you use minikube, you can get its public ip by running `minikube ip`.
+    4. Similarly to `port-forward` solution, you could also use the following command below to submit jobs to the cluster:
```

Comment:
I think it is about point 3 of the previous `* kubectl port-forward:` option where you can also use cli but with `localhost` instead of `<public-node-ip>`